src/types/cell_popovers.ts

```diff
@@ -42,9 +49,9 @@ type OpenCellPopover<C extends ComponentConstructor> = {
 
 export type CellPopoverComponent<
   C extends MaxSizedComponentConstructor = MaxSizedComponentConstructor
-> = ClosedCellPopover | OpenCellPopover<C>;
+> = ClosedCellPopover | OpenCellPopoverComponent<C>;
```

Comment:
Another example of the type naming mix. Here we have a XPopoverComponent that can be a YPopover or a ZPopoverComponent. From this line, I understand that `ClosedCellPopover` is of the same kind of `OpenCellPopoverComponent`, while I would first thing that it should be something similar to `OpenCellPopover`, which is not the case. This is probably just some nitpick and/or a stupid comment, but I was thinking of it